index.dd

```diff
@@ -93,28 +93,36 @@ $(DIVC boxes,
                 Adam D. Ruppe's weekly newsletter. $(SPANC twid)
             )
         )
+        $(TOUR graduation-cap, Learn,
+            $(P Quick start with $(LINK2 http://tour.dlang.org,
+                Introduction to D), explore
+                $(LINK2 $(ROOT_DIR)overview.html, major features) in D,
+                read coming from $(LINK2 $(ROOT_DIR)ctod.html, C) or
+                $(LINK2 $(ROOT_DIR)cpptod.html, C++),
+                setup your $(LINK2 https://wiki.dlang.org/IDEs, IDE) or
+                $(LINK2 https://wiki.dlang.org/Editors, editor),
+                and ask questions in the
+                $(LINK2 https://forum.dlang.org/group/digitalmars.D.learn,
+                    Learn forum).
+            )
+
+            $(P For a deeper dive into D
+            	check out $(LINK2 https://wiki.dlang.org/Books, books) about D, e.g.
+            	Ali Çehreli's
+                $(LINK2 http://ddili.org/ders/d.en/index.html,
+                <cite>Programming in D</cite>) for beginners.
```

Comment:
> "Programming in D is an extensive tutorial by Ali Çehreli and a great starting point for absolute beginners. It doesn't assume that you know any other programming languages, and it also teaches you about some fundamental concepts of programming."

Yeah, I think I wrote that. I didn't mean to say that it's only for newbies, but that it's suitable for newbies. But maybe it is actually aimed mostly at newbies.

I'd go without "for beginners", but I don't mind if it stays.